packages/clerk-js/src/ui/components/OrganizationSwitcher/UserInvitationSuggestionList.tsx

```diff
@@ -120,7 +120,7 @@ const InvitationPreview = withCardStateProvider(
         })}
       >
         <OrganizationPreview
-          elementId='organizationSwitcher'
+          elementId='organizationSwitcherTrigger'
```

Comment:
I'd expect it would be `organizationSwitcherListedOrganization`

Reply:
Done!